DualCameraDemo/DualCameraDemo/DualCameraDemoApp.swift

```diff
@@ -6,9 +6,20 @@ struct DualCameraDemoApp: App {
     
     let dualCameraController = {
         // for now, during development simple way to swap between video recorders
-        let useReplaykitRecorder = false
+        let useReplaykitRecorder = true
```

Comment:
move this to some kind of ui config so we can a) tell which we are currently using and b) toggle between diff types